content/en/docs/gke/private-clusters.md

```diff
@@ -1,307 +1,247 @@
 +++
 title = "Securing Your Clusters"
```

Comment:
Hey @jlewi, I ran "private GKE" in the search bar:

<img width="800" alt="image" src="https://user-images.githubusercontent.com/19637339/93804335-104ee800-fc3e-11ea-82ac-e4e9bfaadca8.png">

I think maybe we could improve the UX by amending the title a little bit.

```diff
- title = "Securing Your Clusters"
+ title = "Securing Your Clusters with Private GKE"
description = "How to secure Kubeflow clusters using private GKE"
```

or

```diff
- title = "Securing Your Clusters"
+ title = "Securing Your Clusters on Private GKE"
description = "How to secure Kubeflow clusters using private GKE"
```

It would be similar to:

```
title = "Kubeflow On-premises on Anthos GKE"
```

LMKWYT

Reply:
Sounds good to me. I'll merge this PR first because the original one is out of date